### ISSUE TITLE
Namespace 'google.maps.places' has no exported member 'AutocompleteSessionToken'

### ISSUE THREAD

**Issue by jaradehr1** [vlafranca/ngxAutocomPlace#13]
Getting the following error when I run >> npm start

```
Error: node_modules/ngx-autocom-place/lib/ngx-autocom-place.service.d.ts:6:38 - error TS2694: Namespace 'google.maps.places' has no exported member 'AutocompleteSessionToken'.

6     sessionToken: google.maps.places.AutocompleteSessionToken;
```

**Comment by jaradehr1** [comment]
Make sure to import google types properly in your code.

**Comment by jaradehr1** [comment]
Also, as a side note.. please make sure to update the tags on this repo to include angular 9, 10, 11.. I tested the previous version **1.0.4** on angular 11 and it was working perfectly fine.. after the latest changes you published, it started giving this error.

**Comment by jaradehr1** [comment]
AutocompleteSessionToken is not a part of google.maps.places namespace

Instead, you will need to initialize the sessionToken each time the user makes a selection (choose an address)
let sessionToken = new google.maps.places.AutocompleteSessionToken();

**Comment by jaradehr1** [comment]
Here is a [reference](https://dzone.com/articles/implement-and-optimize-autocomplete-with-google-pl)

See **Per Request vs Session Token** section

**Comment by jaradehr1** [comment]
See this implementation [reference](https://dzone.com/articles/implement-and-optimize-autocomplete-with-google-pl)

Under section **Per Request vs Session Token**

**Comment by jaradehr1** [comment]
![image](https://user-images.githubusercontent.com/7070770/106413172-a671f680-6406-11eb-8e9a-35e4439c38da.png)

**Comment by vlafranca** [comment]
Sorry about that I forgot to update the googlemaps @types version ...
Fixed in 1.0.6 https://github.com/vlafranca/ngxAutocomPlace/commit/a083a1c4d0d5896830b1e388285ec2245e7737f4
